src/storage/swiftOS.py

```diff
@@ -224,30 +224,34 @@ def create_upload_form(self,sourcepath,containername,prefix,ttl,max_file_size):
         signature = hmac.new(secret, hmac_body, sha1).hexdigest()
 
         # added OBJECT_PREFIX as dir_[task_id] in order to become unique the upload instead of user/filename
-        command = "curl -i {swift_url}/{swift_api_version}/{swift_account}/{containername}/{prefix}/" \
-              " -X POST " \
-              "-F max_file_size={max_file_size} -F max_file_count={max_file_count} " \
-              "-F expires={expires} -F signature={signature} " \
-              "-F redirect={redirect} -F file=@{sourcepath} ".format(
-            swift_url=swift_url, swift_api_version=swift_version, swift_account=swift_account,
-            containername=containername, prefix=prefix, max_file_size=max_file_size,
-            max_file_count=max_file_count,
-            expires=expires, signature=signature, redirect=redirect, sourcepath=sourcepath)
+        command = f"curl -i {swift_url}/{swift_version}/{swift_account}/{containername}/{prefix}/" \
```

Comment:
I'm not familiar with the storage workflow yet, but I guess someone should be doing sanitization of the parameters building a curl command.

Reply:
Yes, `swift_url`, `swift_version`, `swift_account` are ser via environment variables.
In the case of `containername` is taken from the user name in the token, and `prefix` from the task ID.